src/app/log-tool/data-setup/data-setup.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="d-flex flex-column h-100 w-100">
+
+    <nav class="hide-print hidden" routerLinkActive="show" [routerLinkActiveOptions]="{exact:false}">
+        <div class="d-flex tabs subtabs progress-tabs flex-wrap log-tool">
+            <div class="fixed-tab-size" routerLinkActive="active" [routerLinkActiveOptions]="{exact:true}">
+                <a [routerLink]="['/log-tool/data-setup/import-data']">
+                    <span class="step-number">1</span> Import Data</a>
+            </div>
+            <div class="fixed-tab-size" routerLinkActive="active" [routerLinkActiveOptions]="{exact:true}">
+                <a *ngIf="isStepFileUploadComplete" [routerLink]="['/log-tool/data-setup/select-header-data']" routerLinkActive="active">
+                    <span class="step-number">2</span>Select Header Row</a>
+                <a *ngIf="!isStepFileUploadComplete" class="disabled">
+                    <span class="step-number">2</span>Select Header Row</a>
+            </div>
```

Comment:
I am 90% confident that applying either a "disabled" attribute or even the "disable" class will disable the routerLinks so you don't need the *ngIf and two links for these.